docs/components/IconSearch.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react'
+import SearchBar from './SearchBar'
+import { css } from 'otion'
+import * as evergreen from 'evergreen-ui'
+
+const { Pane, majorScale, Text } = evergreen
+
+interface Props {}
+
+const Item: React.FC<{ name: string }> = ({ name }) => {
+  const readableName = name.slice(0, name.indexOf('Icon'))
+  // @ts-ignore
+  if (evergreen[name]) {
+    return (
+      <Pane
+        display="flex"
+        justifyContent="center"
+        alignItems="center"
+        flexDirection="column"
+        cursor="pointer"
+        borderRadius={5}
+        paddingY={majorScale(3)}
+        className={css({
+          ':hover': {
+            opacity: 0.8,
```

Comment:
let's not change the opacity and keep it 1 when hover. hover should not dim the target visually